packages/forms/src/UIForm/fields/Code/Code.component.js

```diff
@@ -32,42 +45,79 @@ try {
 			this.props.onFinish(event, { schema: this.props.schema });
 		}
 
+		onKeyDown(event) {
+			if (event.keyCode === keyCode.codes.esc) {
```

Comment:
while i find this feature to be a nice thing, how do we hint users that it is even a possible move ?

Reply:
In fact I was thinking that this is something that will naturally be tried. During our a11y training, the blind instructor was faced to this issue, and he tried lot of things : (shift) tabs, (multiple) esc, SR shortcuts, ...

But if you think it is really needed, we can think about a hidden description div with this (translated instruction), then linnk the textarea with an `aria-describedby`.

Reply:
Added the instruction div